website/docs/usage/v3.md

```diff
@@ -44,7 +44,7 @@ menu:
 
 </Infobox>
 
-### Custom models using any framework {#feautres-custom-models}
+### Custom models using any framework {#features-custom-models}
```

Comment:
lol the amount of stupid typos like this I make is unreal. I swear my stupid loud MacBook keyboard made it worse

![8e8c837c602ceebffcfae1ae668b1596](https://user-images.githubusercontent.com/13643239/90673201-acf52300-e257-11ea-8166-8ed9307a5b32.gif)